components/camel-google-sheets/pom.xml

```diff
@@ -39,13 +39,59 @@
         <componentPackage>org.apache.camel.component.google.sheets</componentPackage>
         <outPackage>org.apache.camel.component.google.sheets.internal</outPackage>
         <camel.osgi.private.pkg>org.apache.camel.component.google.sheets.internal</camel.osgi.private.pkg>
+
+        <citrus.version>2.8.0</citrus.version>
```

Comment:
Here too: maybe we can move the placeholders in parent/POM

Reply:
Done!